src/services/item/plugins/publication/publicationState/types.ts

```diff
@@ -0,0 +1,14 @@
+import { PackedItem } from '../../../ItemWrapper';
+import { ItemValidation } from '../validation/entities/ItemValidation';
+import { ItemValidationGroup } from '../validation/entities/ItemValidationGroup';
+
+type ValidationStatus = Pick<ItemValidation, 'status'>;
+
+export type ItemMetadata = Pick<PackedItem, 'updatedAt' | 'path' | 'type'> &
+  Partial<Pick<PackedItem, 'public'>>;
```

Comment:
Is it the same as :
```suggestion
export type ItemMetadata = Partial<Pick<PackedItem, 'updatedAt' | 'path' | 'type' | 'public'>>;
```

Reply:
Not exactly ! Yours set all the attributes to nullable while mine construct a type like `{ updatedAt, path, type, public? }` where the only nullable property is public. 

If you know a simpler way to achieve that, I'm open to suggestions :).

Reply:
Oh, you're right, I've missed that. I think it is alright then ! Thank you !